fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/plugin/SimpleValueJqwikPlugin.java

```diff
@@ -218,12 +219,19 @@ public JavaStringConstraint generateStringConstraint(ArbitraryGeneratorContext c
 
 		@Override
 		public JavaIntegerConstraint generateIntegerConstraint(ArbitraryGeneratorContext context) {
-			return new JavaIntegerConstraint(
-				BigInteger.valueOf(this.positiveMinNumberValue),
-				BigInteger.valueOf(this.positiveMaxNumberValue),
-				BigInteger.valueOf(this.negativeMinNumberValue),
-				BigInteger.valueOf(this.negativeMaxNumberValue)
-			);
+			BigInteger positiveMin = BigInteger.valueOf(this.positiveMinNumberValue);
+			BigInteger positiveMax = BigInteger.valueOf(this.positiveMaxNumberValue);
+			BigInteger negativeMin = BigInteger.valueOf(this.negativeMinNumberValue);
+			BigInteger negativeMax = BigInteger.valueOf(this.negativeMaxNumberValue);
+
+			Type type = context.getResolvedType();
```

Comment:
To deal with the type `? extends Byte', it is better to do as follows.

`Types.getActualType(context.getResolvedType())`

Reply:
Thank you!
See the commit below.
- bd85340a95f97030b45bbcb4f0acfbf8b75d3e45